com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumeSceneData.cs

```diff
@@ -230,6 +266,45 @@ internal BakingSet CreateNewBakingSet(string name)
             return set;
         }
 
+        internal void CreateNewBakingState(string name)
```

Comment:
Note that a problem on having the baking states  globally like this is that it forces a given amount of states all with the same name for all the sets in the project. 

This might be relatively ok, but we need to think about the implications a bit.